src/exposed_queries.py

```diff
@@ -0,0 +1,174 @@
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from functools import singledispatchmethod
+from typing import Text, Type
+
+from project_management import (
+    Command,
+    Handler,
+    InvalidTransition,
+    IssueID,
+    State,
+)
+from project_management.commands import (
+    CloseIssue,
+    CreateIssue,
+    ReopenIssue,
+    ResolveIssue,
+    StartIssueProgress,
+    StopIssueProgress,
+)
+from project_management.events import (
+    IssueClosed,
+    IssueOpened,
+    IssueProgressStarted,
+    IssueProgressStopped,
+    IssueReopened,
+    IssueResolved,
+)
+from project_management.eventsourcing import Event, EventStore, TEvent
+
+
+@dataclass
+class Issue:
+    id: IssueID
+    version: int = 0
+    _state: State = None
```

Comment:
Mypy since 0.600 introduced a breaking change that enabled `strict optional` checks by default. Namely, if you do not annotate state as `Optional[State]` it will think it's unsafe. For a good reason - this is a famous billion-dollar mistake in Java - all references are nullable by default.

I'd consider changing annotation or not setting initial state to None

Reply:
done